### DISCUSSION TITLE
Standard Information Exchange format/schema with all the Lenders

### DISCUSSION THREAD

**Discussion by Abhilash4fin** [ONDC-Official/ONDC-FIS-Specifications#24] [Ideas]
Ideas:
1. ONDC to send loan applications data in standard schema/format  to various lenders.   [ Standard loan application schema]
2. Lenders to send offers in standard schema/format 
3. ONDC to send response to the lender again in standard schema/format 
4. Loan status communications to ONDC in standard schema/format 

Possibly someone already working on these? Unless these data exchanges are standardised, integrations with lenders is difficult.

**Comment by 92shreyansh** [comment]
@Abhilash4fin As ONDC doesnt act like a switch and not play role in the complete transaction journey it would be difficult to re-format and standardize at ONDC end.(This is the base ONDC construct as its a Network not a Switch like NPCI or Platform)

Saying this these specification is an attempt to standardize these transaction journey and make sure all the Consumer(BAP/Buyer App/Borrower Platform) and Provider(BPP/Seller App/Lender Platform) talk the same language that is ONDC specification.
